src/web/Content/client/dumpling.py

```diff
@@ -962,9 +991,9 @@ def _create_hang_dump(pid, outpath, debuggerpath):
         osStr = platform.system().lower()
         command = ""
         if osStr == 'linux':
-            command = "./" + debuggerpath + " " + pid + " --name " + outpath + "\memdum" + pid + ".dmp"
+            command = "./" + debuggerpath + " " + pid + " --name " + outpath + "\memdump" + pid + ".dmp"
         elif osStr == 'windows':
-            outputpath = outpath + "\memdum" + pid + ".dmp"        
```

Comment:
was anything reading from `memdum` before? If so does it need a rename also

Reply:
Earlier it was used in helix but now we have shifted this part to dumpling only